VRDR/IJEMortality.cs

```diff
@@ -4139,30 +4139,40 @@ public string SPOUSELV
         /// <summary>Spouse's First Name</summary>
         [IJEField(144, 1385, 50, "Spouse's First Name", "SPOUSEF", 1)]
         public string SPOUSEF
+        // TODO: Implement mapping from FHIR record location: DecedentSpouse
         {
             get
             {
-                // TODO: Implement mapping from FHIR record location: DecedentSpouse
+                string[] names = record.SpouseGivenNames;
+                if (names.Length > 0)
+                {
+                    return names[0];
+                }
                 return "";
```

Comment:
Does the return name need to be left justified to fill all 50 characters?

Reply:
This is handled the same as GNAME and MNAME.   So either we change all of them or none of them.